tools/e2etesting/ParseEnvFile.ps1

```diff
@@ -1,5 +1,6 @@
 param(
```

Comment:
I think we don't need this step anymore. I only used it to read the name of the KeyVault. However, the DetermineKeyVaultName.ps1 script uses another approach (by selecting the KeyVault name based on the 'application'-Tag.
The secrets will later be retrieved directly from the KeyVault. We should add a SecretsFilter in the step "Retrieve KeyVault secrets from existing deployment" below.

Reply:
Great catch. I've removed this step and added a secrets filter to that key vault retrieval step.